domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaycontextparameter/DateTimeEntryValidatorImpl.java

```diff
@@ -13,6 +13,9 @@
 @Component
 public class DateTimeEntryValidatorImpl implements DisplayContextParameterValidator {
 
+    Pattern NOT_ALLOWED_CHARACTERS_PATTERN_DATETIME = Pattern.compile("[YDVzOXxZ]");
+    Pattern NOT_ALLOWED_CHARACTERS_PATTERN_DATE = Pattern.compile("[YDahKkHmsSAnNVzOXxZ]");
```

Comment:
Please follow naming conventions, these variables are not static, final.... Then no need to define them in CAPITAL letters

Reply:
They can be static final now following the move. Adjusted, so no need to rename.